.github/workflows/ci.yaml

```diff
@@ -6,5 +6,47 @@ name: ci
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [8, 10, 12, 14, 16]
+        include:
+          - node: 8
+          - npm_script_extra_args: --ignore @opencensus/exporter-stackdriver
+    env:
+      OPENCENSUS_MONGODB_TESTS: 1
+      OPENCENSUS_REDIS_TESTS: 1
+    services:
+      mongo_service:
+        image: mongo
+        ports: ["27017:27017"]
+      redis_service:
+        image: redis
+        ports: ["6379:6379"]
+
     steps:
-      - run: echo "hello world"
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: v1-node-${{ matrix.node }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            v1-node-${{ matrix.node }}-
+            v1-node-
+
+      - name: Print node/npm version
+        run: node --version && npm --version
+
+      # compile runs in prepare
+      - name: Install modules and dependencies, compile
+        run: npm install
+
+      # lint runs in posttest
+      - name: Run unit tests and lint
+        run: npm run test -- ${{ matrix.npm_script_extra_args }}
+      - name: Submit coverage data to codecov.
+        run: npm run codecov -- ${{ matrix.npm_script_extra_args }}
+        if: ${{ success() && matrix.node == '12' }}
```

Comment:
Ideally this would run on newest node, but workaround for https://github.com/codecov/codecov-node/issues/284